lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -290,7 +290,7 @@ sub locate_local_assets {
 
     for my $key (keys %$assetkeys) {
         my $file = locate_asset($assetkeys->{$key}, $vars->{$key}, mustexist => 1);
-        unless ($file) {
+        unless ($file && $key != 'UEFI_PFLASH_VARS') {
```

Comment:
I think it must be `ne` in Perl when comparing strings (instead of `!=`). At least that's what we use everywhere.